config/initializers/_config.rb

```diff
@@ -4,5 +4,6 @@
 
 Horizon.config = {
   basic_auth_user: ENV['BASIC_AUTH_USER'] || 'admin',
-  basic_auth_pass: ENV['BASIC_AUTH_PASS']
+  expected_version_ruby: ENV['EXPECTED_VERSION_RUBY'],
+  expected_version_node: ENV['EXPECTED_VERSION_NODE']
```

Comment:
This might be a better place to set defaults than the `.env.test` file, so tests can be run locally as well as by hokusai.

Reply:
I don't want to set the version in code so that it can change on the fly, the `.env.test` is used just for comparisons in tests.

Reply:
You can still update the configuration, and any tests depending on specific values should be mocking them anyway like:

```ruby
allow(Horizon).to receive(:config).and_return(expected_version_ruby: '...')
```

Any defaults here would just be a fall-back.